lib/Invoice.php

```diff
@@ -79,6 +79,25 @@ class Invoice extends ApiResource
     use ApiOperations\Retrieve;
     use ApiOperations\Update;
 
+    /**
+     * Possible string representations of the billing reason.
+     * @link https://stripe.com/docs/api/invoices/object#invoice_object-billing_reason
+     */
+    const BILLING_REASON_SUBSCRIPTION           = 'subscription';
+    const BILLING_REASON_SUBSCRIPTION_CREATE    = 'subscription_create';
+    const BILLING_REASON_SUBSCRIPTION_CYCLE     = 'subscription_cycle';
+    const BILLING_REASON_SUBSCRIPTION_UPDATE    = 'subscription_update';
+    const BILLING_REASON_SUBSCRIPTION_THRESHOLD = 'subscription_threshold';
```

Comment:
Super nitpick: Looks like it was carried over from before, but minor misalphabetization here ("U" before "T").